task8.py

```diff
@@ -0,0 +1,24 @@
+'''
+8. You are given a list of numbers and your task is to sort the list without any inbuilt method of python.
+ Input:
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+ Output:
+[1, 2, 3, 5, 6, 8, 9, 10]
+'''
+
+numbers=[2, 5, 6, 1, 3, 8, 9, 10]
+
+#shorted list
+for i in range(0,len(numbers)):
+    for j in range(i+1,len(numbers)):
+        if numbers[i] > numbers[j]:
+            numbers[i], numbers[j] = numbers[j], numbers[i]
+print("sorted list is :- ",numbers)
```

Comment:
Please tell me the name of algorithm you have used for sorting the list.

Reply:
swapping

Reply:
a=10
b=20
swapping  two number without third variable

a = a+b
b = a-b
a = a-b

than a=20 b=10

Reply:
I didn't ask for the logic. Would you please tell me which `sorting algorithm` is this? I want name of sorting algorithm.

Reply:
Bubble Sort

Reply:
Selection Sort

Reply:
It is bubble sort. Now implement your code based on selection sort.